notifier/notifier_test.go

```diff
@@ -185,6 +187,54 @@ func TestHandlerSendAll(t *testing.T) {
 	}
 }
 
+func TestHandlerQueuing(t *testing.T) {
```

Comment:
Ah, I was a bit unclear. I'd name this one `TestHandlerRelabel` and the existing on to this name.